packages/aws-cdk/lib/toolkit/cli-io-host.ts

```diff
@@ -163,6 +164,13 @@ export interface CliIoHostProps {
   readonly isCI?: boolean;
 
   /**
+   * In what scenarios should the CliIoHost ask for approval
+   *
+   * @default RequireApproval.BROADENING
+   */
+  readonly requireApproval?: RequireApproval;
```

Comment:
Add `deploy` somewhere in here? Cause it's different than e.g. `destroy`.


```suggestion
  readonly requireDeployApproval?: RequireApproval;
```